### PR TITLE
Slice2

### PR DESCRIPTION
**Pull request by Al3x4**

### CODE REVIEW

**Comment by Al3x4** [comment]
@andrzejp   you had a button with id cta-button. I changed that id into a class so I can use the class for the button in slice 2.  I hope that's ok.

File: styles/main.css [hunk 1]
```diff
@@ -9,19 +9,56 @@ html {
 body {
     font-family: "Open Sans", "Helvetica", sans-serif;
     line-height: 1.5;
+    background: url("../images/atomist-starry-night.jpg") no-repeat center top;
     background-color: #1b2937;
 }
```
Comment (AndrzejProchyra) [inline styles/main.css]: I would suggest adding `background-size: 100% auto;` here to let the background stretch to the full width on viewports larger than 1680px.

**Review by AndrzejProchyra — CHANGES_REQUESTED** [review]
That's great! I've made a small suggestion and we need to work out how to make the pictures in the next section overlap with the background.

**Comment by Al3x4** [comment]
I fixed the background width issue. 

I also reordered the styles int he CSS file so they're in order of slices. ]

the pull request has been automatically updated.

File: styles/main.css [hunk 2]
```diff
@@ -50,129 +52,113 @@ h1+span {
...
 
 #cta .cta-button {
-    margin-top: 50px;
+  margin-top: 50px;
 }
 
-
+}
```
Comment (AndrzejProchyra) [inline styles/main.css]: Oops! An extraneous brace. This stops the next rule from being applied!

**Review by AndrzejProchyra — CHANGES_REQUESTED** [review]
This is good. There's an extraneous brace though that needs to be removed.

**Comment by Al3x4** [comment]
Okie dokie, fixed the extra bracket. should all be fine now.. wanna have a look?

**Review by AndrzejProchyra — APPROVED** [review]

**Comment by AndrzejProchyra** [comment]
There's something odd going on with vertical spacing but this is good for now!

**Pull request merged**